test/cli.js

```diff
@@ -21,6 +20,10 @@ var cliPath = path.join(__dirname, '../cli.js');
 chalk.enabled = true;
 var colors = require('../lib/colors');
 
+Object.keys(colors).forEach(function (key) {
+	colors[key].enabled = true;
+});
+
```

Comment:
We need to turn on colors no matter what while testing. Also in case that Istanbul turning them off